README.md

```diff
@@ -112,7 +112,7 @@ Our visual vocabulary dashboard helps you to select and create various types of
 each chart type, and offers sample Python code to create these charts with [Plotly](https://plotly.com/python/) and
 embed them into a Vizro dashboard.
 
-<a href="https://huggingface.co/spaces/vizro/demo-visual-vocabulary/">
+<a href="https://vizro-demo-visual-vocabulary.hf.space">
```

Comment:
I am always of two minds when deciding which view to take. The other view also has advantages, as you can like it easier, it leads more naturally to our other demos etc. I think it could make sense to leave it that way?

Reply:
I wasn't a fan of that view because I prefer the 100% height option for any dashboard-demo actually 😬. Our dashboards are typically optimized for that view, so with the HF upper tab bar it sometimes results in unnecessary scrolling.

I do see the value of your second point, but since we reference our general examples gallery just above this section and this section is primarily to showcase the visual vocabulary, I would prefer the view that is actually optimized for this dashboard.